src/cpp/session/include/session/SessionUserSettings.hpp

```diff
@@ -237,6 +237,8 @@ class UserSettings : boost::noncopyable
       return *pPref;
    }
 
+   static void syncConsoleColorEnv();
```

Comment:
Is there a reason this setting needs to be static (vs. being invoked as `userSettings().syncConsoleColorEnv()` as is done elsewhere?).